docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
       HOSTNAME: 'http://platform.local'
     ports:
       - 8080:8080
+      - 9229:9229
```

Comment:
Whats the additional port used for?

Reply:
I added a new line in `server-start-dev` in package.json so that it exposes the debugger